health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/DependentFieldsWrapper.js

```
@@ -298,563 +335,627 @@ function DependentFieldsWrapper({
+                    width: "100%",
+                    maxWidth: "100%",
+                }}
+                className={"version-tag"}
+                labelStyle={{ whiteSpace: "normal", wordBreak: "break-word" }}
+            />
+
+            {/* Actions pinned right */}
+            <div style={{ display: "inline-flex", alignItems: "center", gap: "0.75rem", whiteSpace: "nowrap" }}>
+                <div
+                    role="button"
+                    title={editLabel}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)